cudaungappedextender/include/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+# Enable formatting
+gw_enable_auto_formatting("${CMAKE_CURRENT_SOURCE_DIR}")
```

Comment:
There should be no need to have a CMakeLists.txt in the include folder. It should be enough to have this in the `/cudaungappedextender/CMakeLists.txt` or do you experience problems if you remove this file?

Reply:
Good point. I'll have to check. I was misled by the "CMAKE_CURRENT_SOURCE_DIR".